src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -243,77 +418,89 @@ def execute(self):
         rseDict = result['Value']
         for rse in rses:
           try:
-            self.log.info('Setting productionSEshare for %s : %s', rse, rseDict.get(rse, 0))
+            self.log.info('Setting productionSEshare for', '[RSE: %s : Share: %s', rse, rseDict.get(rse, 0))
             client.add_rse_attribute(rse, 'productionSEshare', rseDict.get(rse, 0))
-          except Exception:
-            self.log.error('Cannot create productionSEshare for %s', rse)
+          except Exception as err:
+            self.log.error('Cannot create productionSEshare for RSE:', rse)
       else:
-        self.log.error('Cannot get SEs : %s', result['Value'])
+        self.log.error('Cannot get SEs', result['Message'])
 
       result = Operations().getSections('Shares')
       if result['OK']:
         for dataLevel in result['Value']:
           result = Operations().getOptionsDict('Shares/%s' % dataLevel)
           if not result['OK']:
-            self.log.error('Cannot get SEs : %s', result['Value'])
+            self.log.error('Cannot get SEs:' % result['Message'])
             continue
           rseDict = result['Value']
           for rse in rses:
```

Comment:
The part about the share is very Belle II specific and might not be useful for other collaboration

Reply:
Agree, harmless really. Could be if-then-elsed by a flag in the CS, if need be.